### PR TITLE
cmd/scollector: add version detail to event logs

### PR DESCRIPTION
**Pull request by gbrayut**

### CODE REVIEW

**Comment by alienth** [comment]
lgtm

File: cmd/scollector/service_windows.go [hunk 1]
```diff
@@ -188,7 +188,7 @@ func runService(name string, isDebug bool) {
 		defer elog.Close()
 	}
```
Comment (kylebrandt) [inline cmd/scollector/service_windows.go]: No blank lines in functions

**Comment by mjibson** [comment]
lgtm

File: cmd/scollector/service_windows.go (hunk 1 above)
Comment (alienth) [inline cmd/scollector/service_windows.go]: That wasn't added here, tho. Whitespace cleanup in a different commit?

Comment (gbrayut) [inline cmd/scollector/service_windows.go]: meh... only one in this file.... I'll fix it, squash and merge.

**Pull request merged**